### PR TITLE
[3PT] Add Twitter Tracking Pixel for PHD

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Adds a second Twitter tracking pixel

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [X] Yes &rarr; This should be handled automatically by the bundling.

## What is the value of this and can you measure success?

PHD should also be able track consenting users.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [X] No (?)
- [ ] Yes (please give details)

### Does this change break ad-free?

- [X] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [X] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
     shouldRun: config.get('switches.twitterUwt', false),
     name: 'twitter',
-    insertSnippet,
+    insertSnippet: insertSnippet('nyl43'),
+});
+
+export const twitterUwtPhd: () => ThirdPartyTag = () => ({
+    shouldRun: config.get('switches.twitterUwt', false),
+    name: 'twitter',
+    insertSnipper: insertSnippet('ny4k9'),
```
Comment (sndrs) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js:29]:
```suggestion
    insertSnippet: insertSnippet('ny4k9'),
```

Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js]: Thanks, but actually I rejigged the logic, please see eccd03b.

File: static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js [hunk 2]
```diff
@@ -4,15 +4,18 @@ import config from 'lib/config';
...
-        // Insert Twitter Pixel ID and Standard Event data below
-        // $FlowFixMe
-        twq('init','nyl43');
-        twq('track','PageView');
+    // Insert Twitter Pixel ID and Standard Event data below
+    // $FlowFixMe
+    twq('init', 'nyl43'); // The Guardian ID
+    twq('track', 'PageView');
+
+    twq('init', 'ny4k9'); // PHD ID
```
Comment (ioanna0) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js:17]: nice generalisation if it works inside the same script !
Does it show 2 scripts when running the [twitter pixel helper plugin](https://chrome.google.com/webstore/detail/twitter-pixel-helper/jepminnlebllinfmkhfbkpckogoiefpd)?

Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js:17]: both twitter tracker working

**Review by ioanna0 — APPROVED** [review]

**Pull request merged**